scripts/ddl2cpp

```diff
@@ -198,7 +198,8 @@ ddlType = (
 )
 
 ddlUnsigned = pp.CaselessLiteral("UNSIGNED").setResultsName("isUnsigned")
-ddlWidth = ddlLeft + pp.Word(pp.nums) + ddlRight
+ddlDigits = pp.Combine("," + pp.Word(pp.nums))
```

Comment:
`ddlDigits = "," + pp.Word(pp.nums)`  should do the trick as well and allow for something like 

`NUMERIC(314, 15)`
(note the space after the comma)